### ISSUE TITLE
Update bullseye image to fix critical security vulenrabilities

### ISSUE THREAD

**Issue by uwinkelvos** [debuerreotype/docker-debian-artifacts#172]
Hey dear maintainers,

debian upstream fixed a couple of very high cve score security vulnerabilities on bullseye.
https://www.debian.org/security/2022/dsa-5218
https://www.debian.org/security/2022/dsa-5216

Could you therefore please update the bullseye and possibly other related images?

**Comment by yosifkit** [comment]
Official Images FAQ:

> Though not every CVE is removed from the images, we take CVEs seriously and try to ensure that images contain the most up-to-date packages available within a reasonable time frame
>
> [...]
>
> We strive to publish updated images at least monthly for Debian. We also rebuild earlier if there is a critical security need. Many Official Images are maintained by the community or their respective upstream projects, like Ubuntu, Alpine, and Oracle Linux, and are subject to their own maintenance schedule.
>
> \- > \- https://github.com/docker-library/faq/tree/0ad5fd60288109c875a54a37f6581b2deaa836db#why-does-my-security-scanner-show-that-an-image-has-cves

The Debian images were updated ~16 days ago (https://github.com/docker-library/official-images/pull/13005), so it could still be a couple weeks out before the images are rebuilt.

If you cannon wait for updated images, then update the specific packages to the fixed version in your own Dockerfile `FROM debian:bullseye` (or your dependent image of choice).
```dockerfile
FROM debian:bullseye
RUN set -e; \
    apt-get update; \
    apt-get install -y -V \
      zlib1g=1:1.2.11.dfsg-2+deb11u2 \
      libxslt1.1=1.1.34-4+deb11u1
```

**Comment by uwinkelvos** [comment]
hey @yosifkit: Thx for the reply. I read the section in the readme and to me that read that this case should qualify as a reason to rebuild. the CVE-2022-37434 has a [base score of 9.8 at NVD](https://nvd.nist.gov/vuln/detail/CVE-2022-37434) and although some distributions assigned a lower score (e.g. suse only 8.1), because the consider the exploitability to be harder, this is potentially a very serious issue. As zlib is avery commonly used library, I would consider this "a security need".

TLDR; It would probably be really helpful if you could state very briefly how you asses CVEs and derive a need to rebuild the image from that assesment.

**Comment by ReillyBrogan** [comment]
It would be preferable to get this one out a bit early in order to pick up the 11.5 release as well.

**Comment by tianon** [comment]
Fixed via https://github.com/docker-library/official-images/pull/13132
